data/test_selectors/nlu.yml

```diff
@@ -0,0 +1,247 @@
+    - What is christmas?
+    - Can you come with me please
+    - Yes can you come with me please meet me in [marietta](city) Ohio
+    - Hi his me miranda I love tom cruise can you make it happen
+    - Can you hep fix it
+    - what is the weather ?
+    - Are you sending me things Roy
+
+- intent: faq/oos_request
+  examples: |
+    - make me coffee
+    - play the lottery
+    - I want to know name of the flight?
+    - Can you get me the specific flight information?
+    - Can I give you feedback?
+    - I want to pay the money from paypal
+    - can I pay with credit card
+    - book flight
+    - Deliver a pizza
+    - How long will it take a letter to get from [new milford]{"entity": "city", "role": "from"} pa to [huntingdon]{"entity": "city", "role": "to"} pa
+    - how to implement faq
+    - What would you suggest where to go for a romance places
+    - Can you fyling him here please i love him so much
+    - Can you fyling him here please
+    - Can you ask him please to fyling here do you watch underworld movie he in it
+    - Can you till him that to fyling here to meet me please
+    - Please stop message him to flying here please
+    - Carolina tell shane brolly that please
+
+- intent: faq/placeholder
+  examples: |
+    - and
+    - so what do I do?
+    - what do i do
+    - .
+    - ok so same
+    - then
+    - next
+    - attitude
+    - talk to me
+    - i
+    - something thing
+    - Hmm
+    - Hmm...
+    - I see
+    - hmm
+
+- intent: faq/takeacut
+  examples: |
+    - do you get paid for offsets
+    - so do you get a cut when I buy these?
+    - what do you get out of this
+    - what are you getting from this?
+    - what you get out of it?
+    - how does this benefit you?
+    - what's the catch?
+    - is there a catch?
+    - there must be a catch
+    - what's the business model here?
+    - are you gonna take a cut?
+    - do you make money off this
+    - do you make money if I buy these offsets?
+    - who benefits from this?
+    - do you get some of that?
+    - who is taking a cut
+    - do you get a cut
+    - are you non-profit?
+    - Does the company take any of the money I pay for offsets?
+
+responses:
+  utter_faq/meta_inquire-ask_bot-challenge:
+    - text: Yes, I am a bot! And I'm part of a research project.
+
+  utter_faq/ask_howdoing:
+    - text: I'm great! Thanks for asking.
+
+  utter_faq/meta_inquire_capabilities:
+    - text: I can help you calculate and buy carbon offsets for your flights.
+
+  utter_faq/oos-future_inquire-ask:
+    - text: That sounds like a relevant question, but I don't know the answer.
+
+  utter_faq/oos_inform:
+    - text: Good to know
+
+  utter_faq/oos_inquire:
+    - text: I'm afraid I have no idea
+
+  utter_faq/oos_request:
+    - text: that's not something I can help with
+
+  utter_faq/placeholder:
+    - text: so
+
+  utter_faq/takeacut:
+    - text: No, my creators don't get a cut from the offsets you buy. I'm just here to help!
+
+
+stories:
```

Comment:
Do we need to keep the story inside `nlu.yaml` file, or we can move it into `data/test_selectors/stories.yml`?
I'd personally prefer not to have a story in a file called `nlu.yml`